azure-resource-optimization-toolkit/.ci_skip

```diff
@@ -0,0 +1 @@
+
```

Comment:
We use the Bootstrap_Main.ps1 to configure things like the "Run As" account which I couldn't get to by using the ARM resource provider for azure automation.  This bootstrap runbook has to execute under the Owner role.  After we use it one time for bootstrap, we clean-up credentials and all other runbooks execute under the built in Run As account.  I didn't see a way meet these requirements within the Travis framework so added ci_skip.